[ 2015-01-13 22:28:20.9941 16133/7f2f59788780 agents/Watchdog/Main.cpp:538 ]: Options: { 'analytics_log_user' => 'william', 'cleanup_pidfiles' => 'L3RtcC9wYXNzZW5nZXItc3RhbmRhbG9uZS4xcm83MXByL3RlbXBfZGlyX3RvdWNoZXIucGlk', 'default_group' => 'william', 'default_python' => 'python', 'default_ruby' => '/home/william/.rvm/gems/ruby-2.2.0-preview1/wrappers/ruby', 'default_user' => 'william', 'log_level' => '0', 'max_pool_size' => '6', 'passenger_root' => '/tmp/passenger-standalone.1ro71pr/locations.ini', 'passenger_version' => '4.0.57', 'pool_idle_time' => '300', 'prestart_urls' => 'aHR0cDovLzAuMC4wLjA6MzAwMAA=', 'temp_dir' => '/tmp', 'union_station_gateway_address' => 'gateway.unionstationapp.com', 'union_station_gateway_port' => '443', 'user_switching' => 'false', 'web_server_passenger_version' => '4.0.57', 'web_server_pid' => '16132', 'web_server_type' => 'nginx', 'web_server_worker_gid' => '-1', 'web_server_worker_uid' => '-1' }
[ 2015-01-13 22:28:21.1096 16136/7f2c20a0e780 agents/HelperAgent/Main.cpp:650 ]: PassengerHelperAgent online, listening at unix:/tmp/passenger.1.0.16132/generation-0/request
[ 2015-01-13 22:28:21.2230 16143/7f89ce625780 agents/LoggingAgent/Main.cpp:321 ]: PassengerLoggingAgent online, listening at unix:/tmp/passenger.1.0.16132/generation-0/logging
[ 2015-01-13 22:28:21.2232 16133/7f2f59788780 agents/Watchdog/Main.cpp:728 ]: All Phusion Passenger agents started!
App 16168 stdout: 
App 16168 stdout: 
[ 2015-01-13 22:28:23.9568 16136/7f2c20929700 Pool2/Implementation.cpp:287 ]: Could not spawn process for application /home/william/dev/little_jack: An error occured while starting up the preloader.
  Error ID: 12d4e9c7
  Error details saved to: /tmp/passenger-error-Ztap7a.html
  Message from application: uninitialized constant YAML (NameError)
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/rack-jekyll-0.3.5/lib/rack/jekyll.rb:10:in `initialize'
  config.ru:2:in `new'
  config.ru:2:in `block in <main>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/rack-1.6.0/lib/rack/builder.rb:55:in `instance_eval'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/rack-1.6.0/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:112:in `eval'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:112:in `preload_app'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:28:in `<main>'


[ 2015-01-13 22:28:23.9787 16136/7f2c1e7f5700 agents/HelperAgent/RequestHandler.h:2306 ]: [Client 20] Cannot checkout session because a spawning error occurred. The identifier of the error is 12d4e9c7. Please see earlier logs for details about the error.
App 16207 stdout: 
App 16207 stdout: 
[ 2015-01-13 22:28:29.2809 16136/7f2c20929700 Pool2/Implementation.cpp:287 ]: Could not spawn process for application /home/william/dev/little_jack: An error occured while starting up the preloader.
  Error ID: 02968807
  Error details saved to: /tmp/passenger-error-t8kceL.html
  Message from application: uninitialized constant YAML (NameError)
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/rack-jekyll-0.3.5/lib/rack/jekyll.rb:10:in `initialize'
  config.ru:2:in `new'
  config.ru:2:in `block in <main>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/rack-1.6.0/lib/rack/builder.rb:55:in `instance_eval'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/rack-1.6.0/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:112:in `eval'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:112:in `preload_app'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:28:in `<main>'


[ 2015-01-13 22:28:29.2853 16136/7f2c1e7f5700 agents/HelperAgent/RequestHandler.h:2306 ]: [Client 20] Cannot checkout session because a spawning error occurred. The identifier of the error is 02968807. Please see earlier logs for details about the error.
App 16227 stdout: 
App 16227 stdout: 
[ 2015-01-13 22:28:31.0133 16136/7f2c20929700 Pool2/Implementation.cpp:287 ]: Could not spawn process for application /home/william/dev/little_jack: An error occured while starting up the preloader.
  Error ID: 61f0d3da
  Error details saved to: /tmp/passenger-error-L0p2vA.html
  Message from application: uninitialized constant YAML (NameError)
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/rack-jekyll-0.3.5/lib/rack/jekyll.rb:10:in `initialize'
  config.ru:2:in `new'
  config.ru:2:in `block in <main>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/rack-1.6.0/lib/rack/builder.rb:55:in `instance_eval'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/rack-1.6.0/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:112:in `eval'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:112:in `preload_app'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /home/william/.rvm/gems/ruby-2.2.0-preview1/gems/passenger-4.0.57/helper-scripts/rack-preloader.rb:28:in `<main>'


[ 2015-01-13 22:28:31.0177 16136/7f2c1e7f5700 agents/HelperAgent/RequestHandler.h:2306 ]: [Client 20] Cannot checkout session because a spawning error occurred. The identifier of the error is 61f0d3da. Please see earlier logs for details about the error.
